### DISCUSSION TITLE
what is with these bogus spell check failed runs?

### DISCUSSION THREAD

**Discussion by clovett** [microsoft/winget-pkgs#87054] [General] [answered]
I'm submitting this PR: https://github.com/microsoft/winget-pkgs/pull/87051/checks

But I get runs like this reporting spellcheck failures to me:
https://github.com/clovett/winget-pkgs/actions/runs/3366593799 

It's reporting spelling errors in packages that have nothing to do with me or my PR? 

![image](https://user-images.githubusercontent.com/3336415/199153857-e9fd3b1f-15f9-4d6f-b2e3-1bad94cb697d.png)

**Comment by russellbanks** [comment, marked as answer]
It is a little annoying, I must admit; a quick and easy fix is to disable the action in your fork.

You can do this under actions > spell checking > disable.

**Comment by marco-eckstein** [comment]
I have the same issue. Every time I push master or another branch, I get the notification _Spell Checking workflow run failed for..._. Until a few days ago, I did not have this issue even with an up-to-date fork. Only after I synced my fork again, it appeared.

Pull requests for the failing branches have been accepted anyway, so apparently the issue can be ignored. But Microsoft, please fix it!
